docs/src/man/guide.md

```diff
@@ -345,3 +345,17 @@ See [Hosting Documentation](@ref) for further information on configuring MkDocs
     of the navigation menu.
 
     **`pages`** defines the hierarchy of the navigation menu.
+
+## Rendering arbitrary markdown files
+
+You can use Documenter to preview arbitrary markdown files like this (example on Linux):
+
+```sh
+cd /tmp
+mkdir src/
+ln -s /path/to/README.md src/
+julia -e "using Documenter; makedocs(format = :html)"
```

Comment:
I'm getting an error when running this one-liner:
```
$ julia -e 'using Documenter; makedocs(format = :html)'
ERROR: syntax: incomplete: premature end of input
```
on v0.6.0 / Linux. It seems as if you can't have `using`/`import` statements in the command line input. Are you seeing the same?